sdks/java/container/build.gradle

```diff
@@ -77,7 +77,9 @@ golang {
 import com.github.jk1.license.render.*
 licenseReport {
   outputDir = 'java_third_party_licenses'
-  projects = [project.rootProject] + project.rootProject.subprojects
+  projects = [project,
+              project.rootProject.findProject(':sdks:java:harness'),
+              project.rootProject.findProject(':sdks:java:io:kafka')]
```

Comment:
@mxm @chamikaramj  incidentally, is this Kafka dep in the core Java harness still required?

Reply:
I think this can be removed now that Kafka dependencies are included in "sdks:java:io:expansion-service" which is used and staged by the kafka x-lang wrapper. But have to run some tests to confirm.